Model/Api/Capabilities.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Omise\Payment\Model\Api;
+
+use OmiseCapabilities;
+
+class Capabilities extends Object
```

Comment:
Idea of that functionality is getting capabilities of user.
the api is also called capabilities in omise-php
call is also to backend about capabilities.

Reply:
but capabilities will develop, and when that will get shape I might do refactoring. For now I like to leave it as Capabilities

Reply:
I was intending to have `Model/Api/*` structure to be represented as a db-layer (but fetch from API instead of real DB). Capabilities sounds good to me. 

But the way to execute this class is a bit ambiguous.. 
`Model\Api\Capabilities::read()` > `Model\Capabilities::get()` > then OmiseConfigProvider got `Installment's backend`.
Would be nice to refactor a bit to explicitly shows its intention. (also, I think this layer should not do filter by itself. The filter one should be in `Model\Capabilities` and keep this one simple and does only retrieving data from API job)

Reply:
@guzzilar Re-read the code - the object here is getting the `backends` info from within capabilties object, but it calls itself `capbilities`. This creates confusion we could do without

Reply:
@jonrandy yeah I got that. I mean, this could instead be
```php
// Model\Api\Capabilities
class Capabilities
{
    public function retrieve()
    {
        // ..
    }
}
```

And just that. Just a layer to connect with Omise API to fetch data back to its object. 
Then `Model\Capabilities` can provide a method to filtering backend later on.

Reply:
cc @jacstn guess you missed this conversation.